### PR TITLE
{tools}[GCCcore/11.3.0] nano v6.4

### PR DESCRIPTION
**Pull request by anselmicz**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by anselmicz** [comment]
Test report by @anselmicz
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
debian.local - Linux Debian GNU/Linux 11, x86_64, AMD Ryzen 7 PRO 4750U with Radeon Graphics (zen2), Python 3.9.2
See https://gist.github.com/d53fc8355ba0c8448019b3aef7279b61 for a full test report.

**Comment by anselmicz** [comment]
Test report by @anselmicz
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
login1.barbora.it4i.cz - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 3.6.8
See https://gist.github.com/f044c32be5ac97bb728b87d0320dae69 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/0e5fe41bbecc682a6c328b64ad0630fd for a full test report.

File: easybuild/easyconfigs/n/nano/nano-6.4.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+easyblock = 'ConfigureMake'
+
+name = 'nano'
+version = '6.4'
+
+homepage = 'https://www.nano-editor.org/'
+docurls = 'https://www.nano-editor.org/docs.php'
+description = """a simple editor, inspired by Pico"""
+
+toolchain = SYSTEM
```
Comment (verdurin) [inline easybuild/easyconfigs/n/nano/nano-6.4.eb:13]: @anselmicz Is there any reason not to put this at `GCCcore`? That seems more natural to me.

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by anselmicz** [comment]
Test report by @anselmicz
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
login1.barbora.it4i.cz - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 3.6.8
See https://gist.github.com/0bb23e8465b15cbf408e14fb0db0ba16 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3114.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/8e59b0c5495e92fc2f57ac0b3a97485d for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @anselmicz!

**Pull request merged**